andromeda/inaturalist.py

```diff
@@ -52,17 +53,41 @@ def add_fieldnames(self, new_fieldnames):
         self.fieldnames.extend(new_fieldnames)
 
 
-def get_inaturalist_observations(user_id, add_sat_rgb_data, add_landcover_data):
+def count_observerations(user_id):
+    observations_dict = pyinaturalist.get_observations(user_id=user_id, count_only=True)
+    return observations_dict["total_results"]
+
+
+def get_observations(user_id, limit=None):
+    total_observations = count_observerations(user_id=user_id)
+    if limit:
+        # when using per_page the observations are sorted with the most recent first
+        observations_dict = pyinaturalist.get_observations(user_id=user_id, per_page=limit)
+    else:
+        # when using page="all" the observations are sorted descending
+        observations_dict = pyinaturalist.get_observations(user_id=user_id, page="all")
+    return observations_dict["results"], total_observations
+
+
+def get_label(idx, total_observations, reversed):
+    if reversed:
+        num = total_observations - idx
+    else:
+        num = idx + 1
+    return f"p{num}"
+
+
+def get_inaturalist_observations(user_id, add_sat_rgb_data, add_landcover_data, limit):
```

Comment:
So we're leaving the backend satellite RGB code and just removing checkboxes so that input is fixed as False?

Reply:
Yes. I expect we would allow deploying Andromeda with a CSV in the future. So future QUEST type classes could provide a class specific CSV to augment the data returned.

Reply:
In that case, would it make sense to leave the UI portions in place in comments connected to a note to the effect of "turn these on instead to include sat RGB or other custom CSV add-on"?

Reply:
I didn't think the RGB checkbox and UI changes would be generic enough to just use as is. I could replace the RGB logic to allow the custom CSV to be configurable at deployment time.

Reply:
From my perspective it seems as generic as the checkbox to include or not, the only change would be the text next to the checkbox. It seems helpful to leave, especially as a feature we may add back in some way down the road. (Not that I'm trying to make more work for you!)

Reply:
I'll go ahead and add a way to supply the custom CSV and various labels.